### PR TITLE
Fixed some grammatical errors and typos.

### PR DESCRIPTION
**Pull request by Ash-Shaun**
Done grammatical and typo fixes.

Signed-off-by: Aslam aslamsmuhammed1@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ http://www.openebs.io/
  
 **OpenEBS** allows you to treat your persistent workload containers, such as DBs on containers, just like other containers. OpenEBS itself is deployed as just another container on your host and enables storage services that can be designated on a per pod, application, cluster or container level, including:
 - Data persistence across nodes, dramatically reducing time spent rebuilding Cassandra rings for example.
-- Synchronization of data across availability zones and cloud providers improving availability and decreasing attach / detach times for example.
-- A common layer so whether you are running on AKS, or your baremetal, or GKE, or AWS - your wiring and developer experience for storage services is as similar as possible.
+- Synchronization of data across availability zones and cloud providers improving availability and decreasing attach/detach times for example.
+- A common layer so whether you are running on AKS, or your bare-metal, or GKE, or AWS - your wiring and developer experience for storage services is as similar as possible.
```
Comment (mahebbar) [inline README.md]: we use bare metal across the docs

Comment (Ash-Shaun) [reply README.md]: Should I change bare-metal to bare metal and update the pull request?

Comment (mahebbar) [reply README.md]: Yes please!

Comment (mahebbar) [reply README.md]: I will approve once done

Comment (Ash-Shaun) [reply README.md]: Sure! Working on it

**Comment by Ash-Shaun** [comment]
Made the changes and updated the pull request.

**Pull request merged**